gardenhub/templates/gardenhub/order_form.html

```diff
@@ -18,9 +18,13 @@
           Select a Plot
         </div>
         <div class="menu">
-          {% for plot in form.plot.field.queryset %}
-            <div class="item" data-value="{{ plot.id }}">{{ plot }}</div>
-          {% endfor%}
+          {% if form.plot.field.queryset.count == 1 %}
+            <div class="item" data-value="{{ plot.id }}">{{ form.plot.field.queryset.0.title}}</div>
```

Comment:
Here `plot` is not set. It's only set in the next part because it's within the for loop, so you need to set the `data-value` similar to how you're creating the title.